### PR TITLE
Nav 257 translate api

### PR DESCRIPTION
**Pull request by jsoppela**

### CODE REVIEW

File: tests/test_logic.py [hunk 1]
```diff
@@ -159,6 +160,7 @@ def test_complete_task_updates_workflow_state(empty_workflow_state):
 def test_complete_task_includes_created_time(empty_workflow_state):
     task_id = "Task1"
     now = datetime.now(timezone.utc)
+    sleep(0.1)
```
Comment (tomeksabala) [inline tests/test_logic.py]: Did you have some problems with the test? I think it runs fine locally and with github actions.

File: app.py [hunk 2]
```diff
@@ -27,6 +28,8 @@ def create_app(config_object=None):
             traces_sample_rate=1.0
         )
 
+    babel = Babel(app)
```
Comment (tomeksabala) [inline app.py:31]: don't assign babel to a local variable as you're not using it afterwards, also the following newline is not needed.
Take a lookat how `CORS(app)` is initialized. You have the exact same usecase.

**Review by tomeksabala — CHANGES_REQUESTED** [review]

File: app.py (hunk 2 above)
Comment (jsoppela) [reply app.py:31]: On row 57 of `app.py` the `babel` local variable is used in a decorator to define the `get_locale` function for the locale selector.

File: tests/test_logic.py (hunk 1 above)
Comment (jsoppela) [reply tests/test_logic.py]: Without this addition `test_complete_task_includes_created_time` fails on my local as variables `now` and `actual_created` match exactly. The assertion in this test requires `now` to be less and not equal to `actual_created`. I opted for `sleep` as changing that assertion to accept equal values might be against the intent of this test.

Comment (tomeksabala) [reply tests/test_logic.py]: Ok, I see. That's strage it came up so late in the process. I don't think using sleep here is a good idea though.
How about transform the `now` variable to `second_ago` with help of timedelta
```
second_ago = datetime.now() - datetime.timedelta(seconds=1)
```

File: app.py (hunk 2 above)
Comment (tomeksabala) [reply app.py:31]: Ok, I see. flask-babel has the init_app interface like many other flask extensions. I thought you should use it:
https://flask-babel.tkte.ch/#flask_babel.Babel.init_app
and configure it like we do with db. Then you can have all babel configuration done in a separate file.
But it'll have the same problem of accessing app.config before creating the flask app (a problem we've discussed in nav engine and sql-alchemy localization).

It's not ideal but let's leave it like that. Thanks.

**Review by tomeksabala — CHANGES_REQUESTED** [review]

File: tests/test_logic.py (hunk 1 above)
Comment (jsoppela) [reply tests/test_logic.py]: I've made the recommended change in commit 14501ab49cbf42d17c87a97e445a5034e3949c30

**Review by tomeksabala — APPROVED** [review]

**Pull request merged**